python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -190,6 +183,17 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
                 for j in range(wLenLines.shape[0]):
                     wLenLinesArr[j] = wLenLines[j]
                 lineListPix = drpStella.createLineList(wLenArr, wLenLinesArr)
+
+                log.logging.getLogger("pfs::drp::stella::math::createLineList").setLevel(log.DEBUG)
+                try:
+                    lineListPix = drpStella.createLineList(wLenArr, wLenLinesArr)
+                except:
```

Comment:
Fixed by using `except Exception as e` and then printing out a `log.warn` message showing `e.message`